src/_locales/en_US/messages.json

```diff
@@ -20,6 +20,12 @@
     "disabled_for_these_domains": {
         "message": "Privacy Badger is disabled for the domains listed below. You can add a domain either via the Privacy Badger icon in the address bar or by typing the domain (not a URL) in the box and clicking the \"Add domain\" button."
     },
+    "dnt_noaction": {
```

Comment:
Yes, if we're modifying a key, we need to do it everywhere or we'll end up orphaning existing translations.